src/lib/store/reducer/boardReducer.js

```diff
@@ -53,6 +67,19 @@ const boardReducer = (state, action) => {
           return board;
         }),
       };
+    case MOVE_BOARD_ITEM_ACTION: {
+      const findBoardItem = [...state.boards.map((board) => board.items).flat()].find((item) => item.id === action.payload.nodeId);
```

Comment:
P3: flatMap 써서 간결하게 변경할 수 있을 것 같아요~

```suggestion
      const findBoardItem = [...state.boards.map((board) => board.items).flat()].find((item) => item.id === action.payload.nodeId);
```

Reply:
> P3: flatMap 써서 간결하게 변경할 수 있을 것 같아요~

```js
const findBoardItem = state.boards.flatMap((board) => board.items).find((item) => item.id === action.payload.nodeId);
```

flatMap 안써봤었는데 이런 경우에 많이 단축되네요! 가독성 면에서 좋은것 같습니다. 👍